src/log_to_disk.c

```diff
@@ -106,10 +106,10 @@ void log_to_disk(int from_lan) {
 
     } else {			/* qso from lan */
 
-	/* LOGENTRY contains 82 characters (node,command and logline */
-	g_strlcpy(lan_logline, lan_message + 2, 81);
-	char *fill = g_strnfill(80 - strlen(lan_logline), ' ');
-	g_strlcat(lan_logline, fill, 81);    /* fill with spaces if needed */
+	/* LOGENTRY contains 89 characters (node,command and logline */
+	g_strlcpy(lan_logline, lan_message + 2, LOGLINELEN);
+	char *fill = g_strnfill(LOGLINELEN - 1 - strlen(lan_logline), ' ');
+	g_strlcat(lan_logline, fill, LOGLINELEN);    /* fill with spaces if needed */
 
 	if (cqwwm2) {	    /* mark as coming from other station */
```

Comment:
How about removing this dubious check and have QSOs coming from other nodes marked with an asterisk?
![image](https://github.com/user-attachments/assets/ace016c5-ba59-4930-b49a-ddb46fecd98b)


Or even better: align the working with the man page - 
 CQWW_M2
              Put  the node ID into the logline (just after the QSO number) to
              support Multi/2 operation where the station logging the QSO must
              be in the Cabrillo file.  This can also be used for M/1 and M/M,
              to enable post-contest analysis of the nodes.

Reply:
That is a good idea. But I would like to check first which other places needs to be changed to make it work and change that afterwards.

Reply:
Just reread the CQWW rules about logging the second stations contacts.
<The MULTIPLIER SIGNAL will be designated with a “1” in the transmitter column of the Cabrillo log. >

Reply:
Aha, thanks. Then this will need some work to sort out.